src/Sulu/Bundle/MediaBundle/Command/InitCommand.php

```diff
@@ -26,6 +26,10 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        if (!$this->getContainer()->has('sulu_media.media.storage.local.path')) {
+            return;
```

Comment:
This is not correct as it will not create the format cache folders. 
I personally would remove the whole storage.local part from this command as the LocalStorage itself create the folder also when it does not exist. This was only needed in sulu-standard where the uploaded folder was not in var directory and the permissions needed to set on the uploaded folder.
The format cache folder create is needed as permissions are still need to set on them and you don't want to set them on the whole public directory.